config/machines/gaya.json

```diff
@@ -4,8 +4,8 @@
     "partitions":["production"],
     "prog_environment":"builtin",
     "launch_options":["-bind-to core"],
-    "reframe_base_dir":"/data/home/cladellash/benchmarking/build/reframe",
-    "reports_base_dir":"/data/home/cladellash/benchmarking/reports/",
+    "reframe_base_dir":"$PWD/build/reframe",
+    "reports_base_dir":"$PWD/reports/",
     "input_dataset_base_dir":"/data/scratch/cladellash/feelppdb/input_data",
     "output_app_dir":"/data/scratch/cladellash/feelppdb",
```

Comment:
What does $PWD refer to ?  it is unclear. Below we have yet other PATHs

Reply:
is it possible to find a better name ?

Reply:
$PWD is the linux environment variable containing the path of the current working directory. The name is not chosen by the application. The config files support ANY environment variable in the form `$VARIABLE`